invenio_vocabularies/administration/views/vocabularies.py

```diff
@@ -9,22 +9,20 @@
 
 """Vocabularies admin interface."""
 
-from invenio_administration.views.base import (
-    AdminResourceEditView,
-    AdminResourceListView,
-)
+from invenio_administration.views.base import AdminResourceListView
+from invenio_i18n import lazy_gettext as _
 
 
 class VocabulariesListView(AdminResourceListView):
     """Configuration for vocabularies list view."""
 
     api_endpoint = "/vocabularies/"
     name = "vocabulary-types"
-    menu_label = "Vocabulary Types"
+    menu_label = _("Vocabulary Types")
     resource_config = "vocabulary_admin_resource"
     search_request_headers = {"Accept": "application/json"}
-    title = "Vocabulary Types"
-    category = "Site management"
+    title = _("Vocabulary Types")
+    category = _("Site management")
```

Comment:
is this attribute used to group such views on the administration page? Do you know which attribute is used for that? the problem is, if we would have namespacing and this value is translated differently on different packages the `site managment` views will not more combined together